livekit-agents/livekit/agents/stt/stt.py

```diff
@@ -1,8 +1,26 @@
+from __future__ import annotations
+
 from abc import ABC, abstractmethod
 from livekit import rtc
 from typing import Optional, List
-from dataclasses import dataclass
+from dataclasses import dataclass, field
 from ..utils import AudioBuffer
+from enum import Enum
+
+
+class SpeechEventType(Enum):
+    # indicate the start of speech
+    # if the STT doesn't support this event, this will be emitted as the same time as the first ITERIM_TRANSCRIPT
+    START_OF_SPEECH = 0
+    # interim transcript, useful for real-time transcription
+    INTERIM_TRANSCRIPT = 1
+    # final transcript, emitted when the STT is confident that a certain
+    # portion of speech will not change
+    FINAL_TRANSCRIPT = 2
+    # indicate the end of speech
+    # emitted when the user stops speaking
+    # the first alternative will be a combination of all the received final transcripts
```

Comment:
all of the received final transcripts or interim transcripts?

Reply:
final transcript, I changed the comment a little bit